packages/fx-core/src/plugins/resource/bot/index.ts

```diff
@@ -103,6 +103,24 @@ export class TeamsBot implements Plugin {
     );
   }
 
+  public async generateArmTemplates(context: PluginContext): Promise<FxResult> {
+    Logger.setLogger(context.logProvider);
+
+    const result = await this.runWithExceptionCatching(
+      context,
+      () => this.teamsBotImpl.generateArmTemplates(context),
+      true,
+      LifecycleFuncNames.GENERATE_ARM_TEMPLATES
+    );
+
+    await ProgressBarFactory.closeProgressBar(
```

Comment:
Do we create progress bar during generating arm templates? Seems we didn't in frontend and function plugin.

Reply:
Ok. Will remove this.

Reply:
Fixed.